jsonschema-core.xml

```diff
@@ -288,6 +278,36 @@
                         zeros) are insignificant.
                     </t>
                 </section>
+
+                <section title="Non-JSON Instances">
+                    <t>
+                        It is possible to use JSON Schema with a superset of the JSON Schema data model,
+                        where an instance may be outside any of the six JSON data types.
+                    </t>
+                    <t>
+                        In this case, annotations still apply; but validation keywords will not be useful,
+                        as they will always pass or always fail.
+                    </t>
+                    <t>
+                        A custom vocabulary may define support for a superset of the core data model.
```

Comment:
- does this paragraph belong in the section "Non-JSON Instances"?

- does this paragraph mean that a custom vocabulary may be expressed in terms of another custom vocabulary? Or that this is *not* permitted and the core vocabulary only may be used? (I strongly prefer the latter, as implementations could otherwise be stuck in a recursive chain of "load vocabulary X first to understand vocabulary Y". I'd at least want someone to show an example of this being done (e.g. in an appendix) to demonstrate that it allowed and provide a potential test case for implementations.

Reply:
>  does this paragraph belong in the section "Non-JSON Instances"?

If an implementations adds support for a superset of the data model, that means it's supporting data that cannot be represented by `application/json`. The way for a schema to declare using this superset would be with a custom `$schema` keyword.

> does this paragraph mean that a custom vocabulary may be expressed in terms of another custom vocabulary

I haven't really thought that far out, but that seems like a concern for the vocabularies section.

(I hope I'm understanding the question correctly.)